packages/react-component-library/src/components/List/ListItem.tsx

```diff
@@ -13,7 +13,7 @@ export interface ListItemProps extends ComponentWithClass {
   /**
    * Description text to display for an individual item.
    */
-  children: string | string[]
+  children: string | string[] | React.ReactNode
```

Comment:
I think string | string[] is redundant because it's implicitly included as part of the ReactNode type.